cmd/forklift-controller/main.go

```diff
@@ -113,12 +118,75 @@ func main() {
 		log.Error(err, "unable to add kubevirt export APIs to scheme")
 		os.Exit(1)
 	}
+	if err := promv1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "unable to add Prometheus APIs to scheme")
+		os.Exit(1)
+	}
 	if err := template.Install(mgr.GetScheme()); err != nil {
 		log.Error(err, "proceeding without optional OpenShift template APIs")
 	}
 	if err := instancetype.AddToScheme(mgr.GetScheme()); err != nil {
 		log.Error(err, "proceeding without optional kubevirt instance type APIs")
 	}
+
+	openshift := os.Getenv("OPENSHIFT")
+	if openshift == "" {
+		openshift = "false"
+	}
```

Comment:
we already [have this logic in the settings](https://github.com/kubev2v/forklift/blob/main/pkg/settings/settings.go#L71), how about taking this configuration from there?